src/parsers/perf/perfparser.cpp

```diff
@@ -1145,6 +1151,10 @@ class PerfParserPrivate : public QObject
     {
         Q_ASSERT(string.id == strings.size());
         strings.push_back(QString::fromUtf8(string.string));
+
+        if (string.string == QByteArray("sched:sched_switch")) {
```

Comment:
QByteArrayLiteral or just compare to a string literal? your code forced an unneeded allocation